youtube_dl/extractor/generic.py

```diff
@@ -214,6 +214,24 @@ class GenericIE(InfoExtractor):
             },
             'playlist_mincount': 100,
         },
+        # RSS feed with items description
+        {
+            'url': 'http://radiofrance-podcast.net/podcast09/rss_14631.xml',
+            'info_dict': {
+                'id': 'http://radiofrance-podcast.net/podcast09/rss_14631.xml',
+                'title': 'Certains l\'aiment Fip',
+                'description': 're:.*FIP met le cinéma.*'
+            },
+            'playlist': [{
+                'info_dict': {
+                    'id': '14631-22.11.2020-ITEMA_22491517-2020Y24252S0327',
+                    'ext': 'mp3',
+                    'title': 'Les musiques de Valérie Lemercier',
+                    'description': 're:.*occasion de la sortie.*',
+                    'upload_date': '20201122',
+                },
+            }],
+        },
```

Comment:
Just add description check to http://podcastfeeds.nbcnews.com/audio/podcast/MSNBC-MADDOW-NETCAST-M4V.xml test instead.